pkg/kube/client.go

```diff
@@ -391,16 +392,25 @@ func (c *Client) Update(namespace string, originalReader, targetReader io.Reader
 // Delete deletes Kubernetes resources from an io.reader.
 //
 // Namespace will set the namespace.
-func (c *Client) Delete(namespace string, reader io.Reader) error {
+func (c *Client) Delete(namespace string, reader io.Reader, timeout int64, shouldWait bool) error {
```

Comment:
Just an FYI that this is a breaking API change. Not many people should be using `pkg/kube` directly that much, but if we need to add the timeout, we'll probably need to add another method so as to not break backwards compatibility